.github/workflows/ci.yaml

```
@@ -1,16 +1,240 @@
+            type=semver,pattern={{major}}.{{minor}}
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          registry: "${{ needs.resolve.outputs.registry }}"
+          username: "${{ github.actor }}"
+          password: "${{ secrets.GITHUB_TOKEN }}"
+
+      - name: Create manifest list and push
+        working-directory: /tmp/digests
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2046:warning:1:33: Quote this to prevent word splitting

(shellcheck)